### PR TITLE
Add frequency device class for sensor

### PR DESCRIPTION
**Pull request by epenet**
## Proposed change

Add tamper device class for binary sensor

home-assistant/architecture#600

home-assistant/core#59700

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#59700
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Hedda** [comment]
> _Add **tamper** device class for binary sensor_

Should be "Add **frequency** device class for binary sensor"

**Comment by epenet** [comment]
Where did I make this mistake?

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/sensor.markdown [hunk 1]
```diff
@@ -24,6 +24,7 @@ The type of data a sensor returns impacts how it is displayed in the frontend. T
 - **current**: Current in A.
 - **date**: Date string (ISO 8601).
 - **energy**: Energy in Wh or kWh.
+- **frequency**: Frequency in Hz, MHz or GHz.
```
Comment (frenck) [inline source/_integrations/sensor.markdown]:
```suggestion
- **frequency**: Frequency in Hz, kHz, MHz or GHz.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**